merkletree/testutil.go

```diff
@@ -0,0 +1,39 @@
+package merkletree
+
+import (
+	"testing"
+
+	"github.com/coniks-sys/coniks-go/crypto"
+)
+
+var keyPrefix = "key"
+var valuePrefix = []byte("value")
+
+// StaticPAD returns a pad with a static initial STR for _tests_.
+func StaticPAD(t *testing.T, ad AssocData) *PAD {
```

Comment:
See my comment above regarding `crypto.StaticVRF` and such.

Reply:
done.